client/editSchedule.js

```diff
@@ -1,37 +1,27 @@
-var editingRoundReact = new ReactiveVar(null);
+var eventToEditReact = new ReactiveVar(null);  // TODO I don't think this needs to be reactive - Lars
```

Comment:
I am pretty sure this does need to be reactive, otherwise blaze isn't going to fire over in our template. See https://github.com/larspetrus/ccm/blob/master/templates/editSchedule.html#L43 for where we refer to this reactive variable (`eventToEdit` is the helper).